inference-engine/src/preprocessing/cpu_x86_avx2/ie_preprocess_gapi_kernels_avx2.cpp

```diff
@@ -131,6 +131,331 @@ void calcRowArea_32F(float dst[], const float *src[], const Size& inSz,
     calcRowArea_impl(dst, src, inSz, outSz, yalpha, ymap, xmaxdf, xindex, xalpha, vbuf);
 }
 
+static inline void verticalPass_8U(const uint8_t *src0[], const uint8_t *src1[], uint8_t tmp[],
+                     v_int16& b0, v_int16& b1, v_int16& b2, v_int16& b3,
+                     v_uint8& shuf_mask, int w, int half_nlanes) {
+    v_int16 val0_0 = v_load_ccache_expand(&src0[0][w]);
+    v_int16 val0_1 = v_load_ccache_expand(&src0[1][w]);
+    v_int16 val0_2 = v_load_ccache_expand(&src0[2][w]);
+    v_int16 val0_3 = v_load_ccache_expand(&src0[3][w]);
+
+    v_int16 val1_0 = v_load_ccache_expand(&src1[0][w]);
+    v_int16 val1_1 = v_load_ccache_expand(&src1[1][w]);
+    v_int16 val1_2 = v_load_ccache_expand(&src1[2][w]);
+    v_int16 val1_3 = v_load_ccache_expand(&src1[3][w]);
+
+    v_int16 t0 = v_mulhrs(v_sub_wrap(val0_0, val1_0), b0);
+    v_int16 t1 = v_mulhrs(v_sub_wrap(val0_1, val1_1), b1);
+    v_int16 t2 = v_mulhrs(v_sub_wrap(val0_2, val1_2), b2);
+    v_int16 t3 = v_mulhrs(v_sub_wrap(val0_3, val1_3), b3);
+
+    v_int16 r0 = v_add_wrap(val1_0, t0);
+    v_int16 r1 = v_add_wrap(val1_1, t1);
+    v_int16 r2 = v_add_wrap(val1_2, t2);
+    v_int16 r3 = v_add_wrap(val1_3, t3);
+
+    v_uint8 q0 = v_packus(r0, r1);
+    v_uint8 q1 = v_packus(r2, r3);
+
+    v_uint8 q2 = v_blend_shiftleft<0xCC /*0b11001100*/, 4>(q0, q1);
+    v_uint8 q3 = v_blend_shiftright<0xCC /*0b11001100*/, 4>(q0, q1);
+
+    v_uint8 q4 = v_shuffle_s8(q2, shuf_mask);
+    v_uint8 q5 = v_shuffle_s8(q3, shuf_mask);
+
+    v_uint8 q6 = v256_permute2x128<0x20>(q4, q5);
+    v_uint8 q7 = v256_permute2x128<0x31>(q4, q5);
+
+    vx_store(&tmp[4 * w + 0], q6);
+    vx_store(&tmp[4 * w + 2 * half_nlanes], q7);
+}
+
+static inline void horizontalPass_U8C1(const short clone[], const short mapsx[], uint8_t tmp[], uint8_t *dst[], int x,
```

Comment:
Suggest to swap `U` and `8` as done with other namings

Reply:
Done.